packages/components/src/font-size-picker/style.scss

```diff
@@ -0,0 +1,3 @@
+.components-font-size-picker {
+	display: contents;
+}
```

Comment:
Instead of introducing a new stylesheet, it likely makes more sense to add it next to the pre-existing Emotion styles:

https://github.com/WordPress/gutenberg/blob/2078ac4c6be2d4554ccb91f1cf93d9c39690ef1d/packages/components/src/font-size-picker/styles.ts#L15-L19